tests/paramgrill.c

```diff
@@ -140,6 +179,126 @@ static unsigned longCommandWArg(const char** stringPtr, const char* longCommand)
     return result;
 }
 
+//assume that clock can at least measure .01 second intervals? 
```

Comment:
I believe it's reasonable to assume that `clock_t` offers a resolution of at least 10ms, if not better. And `clock_t` is the "backup solution", `util.h` use better libraries when available.